airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -45,9 +46,15 @@
 public class Jsons {
 
   // Object Mapper is thread-safe
-  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+  private static final ObjectMapper OBJECT_MAPPER = initMapper();
   private static final ObjectWriter OBJECT_WRITER = OBJECT_MAPPER.writer(new JsonPrettyPrinter());
 
+  private static ObjectMapper initMapper() {
+    final ObjectMapper result = new ObjectMapper();
+    result.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
1. if in an object has `additionalProperties=false`, does this override it? or it his just overriding _default_ behavior?
2. shouldn't be be changing the `additionalProperties` field in objects as well (or instead, depending on the answer to the first question)?

Reply:
To @cgardens point. There are 2 parts to allowing unknown properties:
1. Ensure our Json SerDe accepts it
1. Ensure our Json Schema validation accepts it

The PR only address the first point.

@cgardens I don't think your first point matter since `additionalProperties` only makes sense once the model goes through validation.

Reply:
The `additionalProperties` fields seems to influence only on pydantic, not on Jackson SerDe?

I've tried building a docker image of a connector from this branch then run it on top of the #2228 branch and it passes json validation so everything is able to run normally

Reply:
are you saying this change isn't necessary or that this change is what makes this possible? if you know what's going on here then i'm sure it's fine. i'm not fully understanding your last comment though.

Reply:
Yes, these are the changes that make it possible